configuration/items.md

```diff
@@ -731,6 +731,23 @@ If this is the case, you will find those within the documentation of the Binding
 | `trigger-event-string`                                                                        | Trigger | String                | This Profile can be used to link a trigger channel to a String item. The item's state will be updated to the string representation of the triggering event (e.g. `PRESSED`).                                                                                                                                                                                                                                                                                                                  |
 | `transform:<SERVICE>`                                                                         | State   | All                   | Transformation Profiles can be used to transform the item state (instead of only transforming it in the [Sitemap]({{base}}/ui/sitemaps.html)) using the specified transformation service. You can find the documentation of these Profiles within the [documentation of the Transformation Service](/addons/#transform). E.g. [map](addons/transformations/map/#usage-as-a-profile) or [jsonpath](addons/transformations/jsonpath/#usage-as-a-profile)                                        |
 
+##### Implicit Profiles
+
+If an Item does not have an explicitly defined Profile the framework actually uses 'behind the scenes' an implicit profile.
```

Comment:
uses an implicit profile 'behind the scenes'.

Reply:
Fixed